runtime/src/main/java/org/corfudb/runtime/view/SequencerView.java

```diff
@@ -122,27 +124,24 @@ public StreamAddressSpace getStreamAddressSpace(StreamAddressRange streamsAddres
      * @return address space for each stream in the request.
      */
     public Map<UUID, StreamAddressSpace> getStreamsAddressSpace(List<StreamAddressRange> streamsAddressesRange) {
-        try (Timer.Context context = MetricsUtils.getConditionalContext(sequencerNextOneStream)) {
-            StreamsAddressResponse streamsAddressResponse = layoutHelper(e ->
-                    CFUtils.getUninterruptibly(e.getPrimarySequencerClient()
-                            .getStreamsAddressSpace(streamsAddressesRange)));
-            return streamsAddressResponse.getAddressMap();
-        }
+        StreamsAddressResponse streamsAddressResponse = layoutHelper(e ->
+                CFUtils.getUninterruptibly(e.getPrimarySequencerClient()
+                        .getStreamsAddressSpace(streamsAddressesRange)));
+        return streamsAddressResponse.getAddressMap();
```

Comment:
This is a stream range query (scans the bitsets on the sequencer), we should re-add that metric. Actually, there's a bug right now related to bitset queries #2747

Reply:
done